src/bosh-director/spec/unit/errand/runner_spec.rb

```diff
@@ -152,22 +152,6 @@ module Bosh::Director
             end
           end
 
```

Comment:
It seems like it would be better to change the behavior of this test, rather than removing it completely, so that we're testing that we now return the errand_result object rather than the description.